src/TetherContent.js

```diff
@@ -28,8 +28,9 @@ class TetherContent extends React.Component {
     super(props);
 
     this.handleDocumentClick = this.handleDocumentClick.bind(this);
-    this.checkTogglerTarget = this.checkTogglerTarget.bind(this);
     this.toggle = this.toggle.bind(this);
+    this.checkTogglerTarget = this.checkTogglerTarget.bind(this);
```

Comment:
I just removed the tests that check how many times shouldToggle gets called. Without the bindings, shouldToggle gets called 11 times throughout the specs.